### ISSUE TITLE
Failed to create for Facebook iOS SDK

### ISSUE THREAD

**Issue by nixzhu** [unsignedapps/swift-create-xcframework#38]
Can not create an xcframework from Facebook-ios-sdk

**swift-create-xcframework version:** `1.3.0`
**Swift version:**
Apple Swift version 5.3.2 (swiftlang-1200.0.45 clang-1200.0.32.28)
Target: x86_64-apple-darwin20.3.0
**Environment:** Xcode 12.4 and macOS Big Sur 11.2.3

### Checklist
- [✓] If possible, I've reproduced the issue using the `master` branch of this package
- [✓] I've searched for [existing GitHub issues](https://github.com/unsignedapps/swift-create-xcframework/issues)

### Steps to Reproduce

1. git clone https://github.com/facebook/facebook-ios-sdk.git
2. cd facebook-ios-sdk
3. swift create-xcframework FacebookShare --platform ios

### Expected behavior

An xcframework is created properly

### Actual behavior

<HOME>/facebook-ios-sdk/FBSDKCoreKit/FBSDKCoreKit/Internal/Network/FBSDKURLSessionProxyFactory.h:21:9: fatal error: 'FBSDKCoreKit+Internal.h' file not found
#import "FBSDKCoreKit+Internal.h"

** ARCHIVE FAILED **

The following build commands failed:
	CompileC <DerivedData>/Facebook-bddkrwgyumugvmdfxaxpklxrorhb/Build/Intermediates.noindex/ArchiveIntermediates/FacebookShare/IntermediateBuildFilesPath/Facebook.build/Release-iphoneos/FBSDKCoreKit.build/Objects-normal/armv7/FBSDKTokenCache.o <Home>/facebook-ios-sdk/FBSDKCoreKit/FBSDKCoreKit/Internal/TokenCaching/FBSDKTokenCache.m normal armv7 objective-c com.apple.compilers.llvm.clang.1_0.compiler
(1 failure)
Error: xcodebuild exited with a non-zero code: 65

**Comment by bok-** [comment]
Hi @nixzhu, if you run `swift generate-xcodeproj` inside your checkout of the facebook-ios-sdk are you able to build successfully using the generated project?

**Comment by nixzhu** [comment]
@bok- No, the generated Facebook.xcodeproj built to simulator iPhone 12 mini failed: "FBSDKCoreKit+Internal.h" file not found. I think it's a Facebook SDK's issue.
